src/Callable.cpp

```diff
@@ -197,6 +203,8 @@ Callable::FailureFn Callable::check_fcci(size_t argc, const FullCallCheckInfo *a
 }
 
 int Callable::call_argv_checked(size_t argc, const void *const *argv, const QuickCallCheckInfo *actual_qcci) const {
+    internal_assert(defined());
```

Comment:
I think this should be a user_assert with an error message, because a user might actually try to call a default-constructed callable

Reply:
Done